onnxruntime/core/providers/cpu/ml/svmclassifier.h

```diff
@@ -31,28 +31,28 @@ class SVMCommon {
 
   float kernel_dot(const T* A, int64_t a, const std::vector<float>& B, int64_t b, int64_t len, KERNEL k) const {
     float sum = 0.f;
+    float* pA = (float*)&A[a];
```

Comment:
>(float*)&A[a] [](start = 16, length = 13)

Could (float*)(A + a) work?

Reply:
A seems to be a typed ptr. Do we know it is a float?

Reply:
fixed